spec/cluster_spec.rb

```diff
@@ -97,21 +97,21 @@
     describe "#item_enum_chrons" do
       it "collects all item enum_chrons in the cluster" do
         c = described_class.first
-        expect(c.item_enum_chrons).to eq(["3"])
+        expect(c.item_enum_chrons).to eq(["3 "])
```

Comment:
Why the extra space? It feels like we should `strip` the values (before saving?) so we don't have to worry about this stuff. Or am I missing something?

Reply:
Stripping means an enum or a chron can match an enum_chron, which we don't want.

Reply:
If enums or chrons could have spaces in the middle, then a normalized enum_chron could match an enum or chron.

@mwarin believes spaces are switched out with colons in scrub process / normalization

Reply:
@jsteverman  is going to use `\t` as the separator and add a spec to https://github.com/hathitrust/holdings-backend/blob/master/spec/enum_chron_spec.rb to ensure that tabs are stripped out by normalization.